looprpc/client.proto

```diff
@@ -396,6 +396,12 @@ message SwapStatus {
     // Off-chain routing fees
     int64 cost_offchain = 10;
 
+    // Provided last hop
+    bytes provided_last_hop = 16;
```

Comment:
wdyt about just calling these `last_hop` and `outgoing_chan_set`?

Reply:
I was unsure what would be expected as an api user, but I can see that removing `provided` is more consistent with the rest of the api

Reply:
Yes was my thinking exactly, just to be consistent with naming.